### PR TITLE
LPS-182086 [Raylife Ap] Notification List does not appear on Account settings page and Notifications Page

### PR DESCRIPTION
**Pull request by eduardo-diniz**
[LPS-182086](https://issues.liferay.com/browse/LPS-182086)

### CODE REVIEW

**Comment by eduardo-diniz** [comment]
ci:test:sf

File: modules/apps/site-initializer/site-initializer-raylife-ap/src/main/resources/site-initializer/layouts/11-account-settings/page-definition.json [hunk 1]
```diff
@@ -57,6 +57,19 @@
...
 								"definition": {
 								},
+								"pageElements": [
+									{
+										"definition": {
+											"widgetInstance": {
+												"widgetConfig": {
+													"properties": "route=notification-sidebar"
+												},
+												"widgetName": "com_liferay_client_extension_web_internal_portlet_ClientExtensionEntryPortlet_20096_RAYLIFEAGENTPORTAL001"
```
Comment (joanagrego) [inline modules/apps/site-initializer/site-initializer-raylife-ap/src/main/resources/site-initializer/layouts/11-account-settings/page-definition.json:67]:
```suggestion
											"widgetName": "[$CLIENT_EXTENSION_ENTRY_ID:RAYLIFEAGENTPORTAL$]"
```

File: modules/apps/site-initializer/site-initializer-raylife-ap/src/main/resources/site-initializer/layouts/10_notifications-list/page-definition.json [hunk 2]
```diff
@@ -57,6 +57,19 @@
...
 								"definition": {
 								},
+								"pageElements": [
+									{
+										"definition": {
+											"widgetInstance": {
+												"widgetConfig": {
+													"properties": "route=notification-sidebar"
+												},
+												"widgetName": "com_liferay_client_extension_web_internal_portlet_ClientExtensionEntryPortlet_20096_RAYLIFEAGENTPORTAL001"
```
Comment (joanagrego) [inline modules/apps/site-initializer/site-initializer-raylife-ap/src/main/resources/site-initializer/layouts/10_notifications-list/page-definition.json:67]:
```suggestion
												"widgetName": "[$CLIENT_EXTENSION_ENTRY_ID:RAYLIFEAGENTPORTAL$]"
```

**Review by joanagrego — CHANGES_REQUESTED** [review]

**Comment by eduardo-diniz** [comment]
ci:test:sf

**Comment by joanagrego** [comment]
ci:forward

**Pull request closed without merging**